fcos/release-checklist.md

```diff
@@ -74,6 +74,28 @@ At this point, Cincinnati will see the new release on its next refresh and creat
 
 - [ ] Wait for all releases that will be released simultaneously to reach this step in the process
 - [ ] Go to the [rollout workflow](https://github.com/coreos/fedora-coreos-streams/actions/workflows/rollout.yml), click "Run workflow", and fill out the form
+<details>
+<summary>Rollout general guidelines</summary>
+
+When setting a "Rollout start time" ask "when would be the best time to react to 
+any errors or regressions from updates?"
+Commonly we select 2PM UTC so that the rollout's start at 10am EST, but these can
+be fluid and adjust after talking with the fedora-coreos IRC.
+
+Additionally this is impacted by the day of the week or holidays, again ask 
+"when would be the best time to react to any errors or regressions from updates?"
+
+Commonly we try and release by Tuesday with 48h rollout timelines to insure
+updates are done before the weekend. Consequently the later in the week the release
+the more likely the rollout time allocation will need to shrink or the release will
+need to be pushed. 
+
+A final consideration is a release rebase, i.e. f37 to f38, when this happens its best
+to follow the common case.
```

Comment:
If we're following the common case, is there value to explicitly calling this out as a special case?

Reply:
Great point! What I wanted to express was how the process becomes more ridgid.